src/BalanceTx/Collateral.purs

```diff
@@ -5,7 +5,6 @@ module BalanceTx.Collateral
       )
```

Comment:
I moved these functions from `QueryM` down to `Effect` so that we can use `selectCollateral` directly in `privateKeysToKeyWallet`. This was necessary due to module import cycles with `QueryM`.